app/views/nics.scala.html

```diff
@@ -48,7 +48,7 @@ <h1>
     </h1>
   <p class="lede">@Html(Messages("ats.nics.tax_and_nics.lede"))</p>
     <p>@Html(Messages("ats.nics.tax_and_nics.description",
-      "<span id=\"total-cg-tax-rate\">"+viewModel.nicsAndTaxRate.percent+"</span>"))</p>
+      "<span id=\"total-cg-tax-rate\">"+positiveOrZero(viewModel.nicsAndTaxRate).percent+"</span>"))</p>
 
```

Comment:
same as above we wont need this view change as well if we change the backend code instead.